cms/djangoapps/contentstore/views/videos.py

```diff
@@ -328,6 +331,13 @@ def videos_post(course, request):
     for req_file in req_files:
         file_name = req_file["file_name"]
 
+        # Check if file_name contains non-ascii characters.
+        try:
+            file_name.encode('ascii')
+        except UnicodeEncodeError as exception:  # pylint: disable=unused-argument
+            error_msg = "%s contains non ascii characters in file name." % file_name
```

Comment:
@mushtaqak For user messages, "non ascii" might not be the most understandable. Would it be equally accurate to refer to "characters other than alphanumeric characters?"